website/docs/r/web_pubsub_hub.html.markdown

```diff
@@ -24,7 +24,7 @@ resource "azurerm_user_assigned_identity" "test" {
   location            = azurerm_resource_group.example.location
 }
 
-resource "azurerm_web_pubsub" "example" {
+resource "azurerm_web_pubsub" "test" {
```

Comment:
we use example in docs, so could we change this back?

Reply:
@katbyte I have modified it, please check it, thanks.